<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-5A78VGD" ignored="0" name="Login Test" tests="1" failures="1" timestamp="06 Nov 2019 08:20:50 GMT" time="30.4" errors="0">
  <testcase name="LoginTest" time="13.445" classname="com.NewtourExcel.NewtoursLogin">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.NewtourExcel.NewtoursLogin.LoginTest(NewtoursLogin.java:52)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Login Test -->
